main.py

```diff
@@ -1821,7 +1821,7 @@ async def rarities(message: discord.Interaction):
     s = sum(type_dict.values())
     for k, v in type_dict.items():
         icon = get_emoji(k.lower() + "cat")
-        embed.add_field(name=f"{icon} {k}", inline=True, value=f"Rarity: {round(v / s * 100, 2)}%\nValue: {int(s / v)}")
+        embed.add_field(name=f"{icon} {k}", inline=True, value=f"Rarity: {round(v / s * 100, 2)}%\nValue: {round(s / v)}")
```

Comment:
pretty sure this doesn't matter lol

Reply:
`int()` chops off fractional part, `round()` rounds it based on math rules
yes, this will result in +-1 difference which is not that important but `round()` is more accurate

Reply:
floor gaming

Reply:
me when im in the most similar to `math.floor()` competition and my opponent is `int()`:

Reply:
`int()` truncates, whereas `math.floor()` returns the greatest integer less than or equal to the input
it's the same for positive numbers, but not for negative numbers
this does not affect cat bot whatsoever; I just felt like replying

Reply:
so `int(69.42)` is 69 because 42 gets chopped off and `math.floor(69.42)` is 69 because 69.42 gets turned into an integer

Reply:
and what is the difference in negative numbers

Reply:
![Screenshot_20240615_124542_Termux.jpg](https://github.com/milena-kos/cat-bot/assets/94813389/fde84a42-1c79-4ecc-87d4-c11c0cac01ec)

Reply:
cursed asf screw math.floor
also termux lol

Reply:
yes termux I was on my phone lol
idk what the font is tbh

Reply:
install arch Linux on it
my install of termux was literally 11gb but I deleted it

Reply:
I've already installed ubuntu lol
I've reinstalled termux a few times because of how much random stuff I had installed lol
currently 8.45 gb

Reply:
ubuntu smh atleast use debian

Reply:
what's wrong with ubuntu

Reply:
have you ever met people that use arch linux and they say arch is better than everything

I'm saying debian is better than ubuntu

Reply:
no not really

yeah idk I'm not gonna use either on my phone anyways (why does samsung in the US not let you change OS)

I need to switch to a linux distro I still use windows for some reason and I got like 10 green screens of death yesterday lol

Reply:
whatever debian is better than ubuntu

Reply:
ok